### PR TITLE
Adjusts CGI harvester to use COSMIC search

### PR DESCRIPTION
**Pull request by grmayfie**
CGI data lacked end coordinates, previously. This was due to parsing genomic information from the  'gDNA' field of the CGI file rather than checking for location information in COSMIC. 

This alters the CGI harvester to check for location information in COSMIC first; if no hits are returned, then the previous internal parsing function is used instead, with no changes the expectation that later in the harvest, checks against myvariant.info will grab the additional information. 

@bwalsh @jgoecks

### CODE REVIEW

File: harvester/cgi_biomarkers.py [hunk 1]
```diff
@@ -107,25 +114,52 @@ def split_gDNA(gDNA):
...
+    indiv_mut = evidence['individual_mutation']
+    if len(gDNA) != 0 or len(indiv_mut) != 0:
+        if not LOOKUP_TABLE:
+            LOOKUP_TABLE = cosmic_lookup_table.CosmicLookup(
+                "./cosmic_lookup_table.tsv")
+        if len(indiv_mut) != len(gDNA):
+            print "WE HAVE A PROBLEM!!!"
+            print gDNA
+            print indiv_mut
+            sys.exit()
```
Comment (bwalsh) [inline harvester/cgi_biomarkers.py]: Should this exit be in the code?

**Comment by ahwagner** [comment]
Does this PR conflict with #100 @mayfielg?

**Comment by bwalsh** [comment]
@mayfielg we should be targeting v0.10

**Review by bwalsh — CHANGES_REQUESTED** [review]
we should be targeting v0.10

**Comment by grmayfie** [comment]
Moved to v0.10. Removed excess print lines.

**Pull request merged**